apps/console/src/components/pages/protected/standards/standard-details-accordion.tsx

```diff
@@ -0,0 +1,165 @@
+  }
+
+  const toggleSection = (section: string) => {
+    setOpenSections((prev) => (prev.includes(section) ? prev.filter((s) => s !== section) : [...prev, section]))
+  }
+
+  return (
+    <>
+      <Accordion type="multiple" className="w-full">
+        <div className="flex justify-between">
+          <h2 className="text-2xl">Domains</h2>
+          <Input
+            value={searchQuery}
+            name="standardSearch"
+            placeholder="Search ..."
+            onChange={(e) => setSearchQuery(e.target.value)}
+            icon={<SearchIcon size={16} />}
+            iconPosition="left"
+            variant="searchTable"
+            className="!border-brand"
+          />
+        </div>
+        {Object.entries(groupedControls).map(([category, controls]) => {
+          const isOpen = openSections.includes(category)
+          return (
+            <AccordionItem key={category} value={category}>
+              <AccordionTrigger className="flex items-center gap-2 text-lg font-semibold w-full p-4 cursor-pointer rounded-lg" onClick={() => toggleSection(category)}>
+                <span>{category}</span>
+                {isOpen ? <ChevronDown size={22} className="text-brand" /> : <ChevronRight size={22} className="text-brand" />}
+              </AccordionTrigger>
+              <AccordionContent>
+                <div className="bg-card py-3 rounded-lg border">
+                  <Table>
+                    <TableHeader>
+                      <TableRow>
+                        <TableHead className="w-10">
+                          <Checkbox checked={selectedControls.length === controls.length} onCheckedChange={(checked: boolean) => setSelectedControls(checked ? controls.map((c) => c.id) : [])} />
+                        </TableHead>
+                        <TableHead className="whitespace-nowrap">Ref Code</TableHead>
+                        <TableHead>Description</TableHead>
+                        <TableHead>Subdomain</TableHead>
+                        <TableHead className="whitespace-nowrap">Mapped Categories</TableHead>
+                        <TableHead># of Sub controls</TableHead>
+                      </TableRow>
+                    </TableHeader>
+                    <TableBody>
+                      {controls.map((control) => (
+                        <TableRow key={control.id}>
+                          <TableCell>
+                            <Checkbox checked={selectedControls.includes(control.id)} onCheckedChange={() => toggleSelection(control.id)} />
+                          </TableCell>
+                          <TableCell className="text-blue-400 whitespace-nowrap">{control.refCode}</TableCell>
+                          <TableCell>{control.description}</TableCell>
+                          <TableCell>{control.subcategory}</TableCell>
+                          <TableCell>{control.category}</TableCell>
```

Comment:
This should be the mapped category field, not category
```suggestion
                          <TableCell>{control.mappedCategories}</TableCell>
```